chapters/hardware-software-interface/arrays/drills/tasks/courses/solution/.gitignore

```
@@ -1,2 +1 @@
-courses
-courses.o
\ No newline at end of file
+courses
```

```suggestion
courses

```
Add newlines at the ends of all `.gitignore`s.